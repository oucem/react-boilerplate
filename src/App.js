import React from "react";
import styles from './App.styles.css';

const App = () => {
    return (
        <div className={styles.app}>
            Welcome to the app!
        </div>
    )
};

export default App;