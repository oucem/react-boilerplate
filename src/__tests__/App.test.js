import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

describe('the app component', () => {
    it('should render correctly', () => {
        const component = renderer.create(<App />).toJSON();
        expect(component).toMatchSnapshot();
    })
})
