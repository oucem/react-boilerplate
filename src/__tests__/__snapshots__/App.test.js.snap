// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the app component should render correctly 1`] = `
<div>
  Welcome to the app!
</div>
`;
